presto-hive/src/main/java/com/facebook/presto/hive/ColumnarTextHiveRecordCursor.java

```diff
@@ -545,6 +551,51 @@ else if (type.equals(VARBINARY)) {
         nulls[column] = wasNull;
     }
 
+    private void parseCharColumn(int column)
```

Comment:
It looks like this is a copy of `parseStringColumn`, I think it would be better to just change `parseStringColumn` like you did in the generic reader